src/cpuid/src/intel/leaves.rs

```diff
@@ -0,0 +1,736 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+#![allow(clippy::similar_names, clippy::module_name_repetitions)]
+use std::fmt;
+
+use bit_fields::Equal;
+
+#[allow(clippy::wildcard_imports)]
+use super::registers::*;
+use crate::Leaf;
+
+/// Cache and TLB information keywords.
+#[allow(clippy::non_ascii_literal)]
+static KEYWORDS: phf::Map<u8, &'static str> = phf::phf_map! {
```

Comment:
All this functionality seems related to `Leaf2` but as far as I can see `Leaf2` is not used anywhere in the current implementation. 
Is this maybe a leftover?

Reply:
`Leaf2` is used with `index_leaf!(0x2, Leaf2, IntelCpuid);` which allows accessing it in the `IntelCpuid`.

Reply:
This may be useful in the future, however initial releases of this functionality will be in quite a raw numeric format. Any descriptors and metadata will need to be looked up in specifications published by the CPU vendors. Given that users dealing with CPU configuration will need to use these specifications anyway, I think this is a fair expectation.

Reply:
I've removed it.